requirements/prod.in

```diff
@@ -25,6 +25,6 @@ psycopg2-binary==2.8.4
 pyOpenSSL==19.1.0
 sentry-sdk==0.13.5
 service_identity==18.1.0
-https://github.com/SFDO-Tooling/sfdo-template-helpers/archive/v0.9.0.tar.gz
+https://github.com/SFDO-Tooling/sfdo-template-helpers/archive/master.tar.gz
```

Comment:
Seems safest to keep this pinned so that builds are consistent.

Reply:
Can you tag v0.10.0 then? There are changes in master that make this work: https://github.com/SFDO-Tooling/sfdo-template-helpers/compare/v0.9.0...master

Reply:
Oh I see. I'll do that soon; this is fine for now.